doc/source/user_guide/copy_on_write.rst

```diff
@@ -0,0 +1,183 @@
+.. _copy_on_write:
+
+{{ header }}
+
+*******************
+Copy-on-Write (CoW)
+*******************
+
+.. ipython:: python
+    :suppress:
+
+    pd.options.mode.copy_on_write = True
+
+Copy-on-Write was first introduced in version 1.5.0. Starting from version 2.0 most of the
+optimizations that become possible through CoW are implemented and supported. A complete list
+can be found at TODO
+
+We expect that CoW will be enabled per default in version 3.0
```

Comment:
```suggestion
We expect that CoW will be enabled by default in version 3.0
```

Reply:
Also in this section would be good to highlight the quick benefits of CoW

1. More predictable behavior
2. Improved performance through deferred copying

Reply:
Added